eclair-core/src/main/scala/fr/acinq/eclair/db/migration/CompareChannelsDb.scala

```diff
@@ -0,0 +1,80 @@
+package fr.acinq.eclair.db.migration
+
+import fr.acinq.eclair.BlockHeight
+import fr.acinq.eclair.channel.{DATA_CLOSING, DATA_WAIT_FOR_FUNDING_CONFIRMED}
+import fr.acinq.eclair.db.migration.CompareDb._
+import fr.acinq.eclair.wire.internal.channel.ChannelCodecs.stateDataCodec
+import scodec.bits.ByteVector
+
+import java.sql.{Connection, ResultSet}
+
+object CompareChannelsDb {
+
+  private def compareChannelsTable(conn1: Connection, conn2: Connection): Boolean = {
+    val table1 = "local_channels"
+    val table2 = "local.channels"
+
+    def hash1(rs: ResultSet): ByteVector = {
+      val data = ByteVector(rs.getBytes("data"))
+      val data_modified = stateDataCodec.decode(data.bits).require.value match {
+        case c: DATA_WAIT_FOR_FUNDING_CONFIRMED => stateDataCodec.encode(c.copy(waitingSince = BlockHeight(0))).require.toByteVector
+        case c: DATA_CLOSING => stateDataCodec.encode(c.copy(waitingSince = BlockHeight(0))).require.toByteVector
+        case _ => data
+      }
```

Comment:
Because of this:
https://github.com/ACINQ/eclair/blob/a5bfde7b18a7db23025d2148cc2eee304c8f75e5/eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version0/ChannelCodecs0.scala#L358